_posts/2021-04-22-virtual-workshop-update.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: 2021 US-RSE Virtual Workshop Update
+tags: [events, virtual workshop]
+posted_by: Julia Damerow, Sandra Gesing
+---
+
+As we are preparing for the workshop, we would like to share a few updates.
+
+As our previous meetings and community calls, the virtual workshop will be held on Zoom. Registration information will be made available as we get closer to the workshop date. Please see [this page](https://us-rse.org/virtual-workshop-2021/participate/) for information about participation if you are interested in joining the workshop and/or serving as a reviewer. All workshop participants are expected to abide by the [US-RSE Code of Conduct](https://us-rse.org/code-of-conduct/) at all times before, during, and after the workshop.
+
+**Virtual Workshop Website**
+
+We have set up a dedicated website for the 2021 Virtual Workshop [here](https://us-rse.org/virtual-workshop-2021/). We will update this website as new information becomes available, such as the agenda or registration information.
+
+**Abstract submission via EasyChair**
+
+Abstracts can now be submitted via [EasyChair](https://easychair.org/my/conference?conf=usrsevw2021). If you have never used EasyChair before, you will need to create an account first. Otherwise you should be able to login with your previous account. You can find the Call for Proposals [here](https://us-rse.org/virtual-workshop-2021/participate/). Submission deadline is May 3, 2021, 11:59pm ET.
```

Comment:
```suggestion
Abstracts can now be submitted via [EasyChair](https://easychair.org/my/conference?conf=usrsevw2021). If you have never used EasyChair before, you will need to create an account first. Otherwise you should be able to login with your existing account. You can find the Call for Proposals [here](https://us-rse.org/virtual-workshop-2021/participate/). Submission deadline is May 3, 2021, 11:59 pm ET.
```